python_modules/libraries/dagster-fivetran/dagster_fivetran/translator.py

```diff
@@ -9,6 +10,8 @@
 
 from dagster_fivetran.utils import get_fivetran_connector_table_name, metadata_for_table
 
+MIN_TIME_STR = "0001-01-01 00:00:00+00"
```

Comment:
doesn't look like this gets used anywhere - can we delete this?

Reply:
Done in [7dde6fb](https://github.com/dagster-io/dagster/pull/25911/commits/7dde6fb6421dd9241a26a5f3a4c61731df35f97b)